### ISSUE TITLE
investigate accessibility violations field

### ISSUE THREAD

**Issue by gbinal** [GSA/site-scanning#1453]
It's empty across the board.

**Comment by gbinal** [comment]
this is fixed.  thanks, @luke-at-flexion
